app/views/content/is-teaching-right-for-me/get-school-experience.md

```diff
@@ -42,24 +42,24 @@ During your experience, you’ll get to do things like:
 - observe lessons
 - see how teachers manage a classroom
 - find out how specific subjects are taught
-- speak to teachers and interact with pupils
+- speak to teachers and meet pupils
 - learn more about teacher training - including the application and interview process
 
 ## Find school experience
 
-You can search for and request [school experience](https://schoolexperience.education.gov.uk/) in England.
+Once you've selected chosen a school, you can get more information about their placement availability and any requirements they may have. Some schools may charge fees to cover their costs and you may need to have a DBS check. 
```

Comment:
This doesn't read right, not sure if chosen is there by mistake 'once you've selected a school'